gradle/versions.gradle

```diff
@@ -121,50 +116,44 @@ dependencyManagement {
     dependency 'com.github.arteam:simple-json-rpc-server:1.3'
     dependency 'com.github.arteam:simple-json-rpc-client:1.3'
 
-    dependency 'com.azure:azure-security-keyvault-secrets:4.9.0'
-    dependency 'com.azure:azure-security-keyvault-keys:4.9.0'
-    dependency 'com.azure:azure-identity:1.14.2'
+    dependency 'com.azure:azure-security-keyvault-secrets:4.10.3'
+    dependency 'com.azure:azure-security-keyvault-keys:4.10.3'
+    dependency 'com.azure:azure-identity:1.18.1'
 
-    dependency 'com.zaxxer:HikariCP:6.2.1'
-    dependency 'org.postgresql:postgresql:42.7.7'
+    dependency 'com.zaxxer:HikariCP:7.0.2'
+    dependency 'org.postgresql:postgresql:42.7.8'
 
-    dependencySet(group: 'org.jdbi', version: '3.47.0') {
+    dependencySet(group: 'org.jdbi', version: '3.49.6') {
       entry 'jdbi3-core'
       entry 'jdbi3-sqlobject'
       entry 'jdbi3-testing'
     }
 
-    dependency 'org.flywaydb:flyway-core:11.0.0'
-    dependency 'org.flywaydb:flyway-database-postgresql:11.0.0'
+    dependencySet(group: 'org.flywaydb', version: '11.14.1') {
+        entry 'flyway-core'
+        entry 'flyway-database-postgresql'
+    }
 
-    dependency 'com.google.cloud:google-cloud-secretmanager:2.54.0'
-    dependency 'com.google.protobuf:protobuf-java:3.25.5' // fixes CVE-2024-7254 - transitive from google-cloud-secretmanager
+    dependency 'com.google.cloud:google-cloud-secretmanager:2.77.0'
 
-    dependency 'io.zonky.test.postgres:embedded-postgres-binaries-bom:17.2.0'
-    dependency 'io.zonky.test:embedded-postgres:2.1.0'
+    dependency 'io.zonky.test.postgres:embedded-postgres-binaries-bom:17.6.0'
+    dependency 'io.zonky.test:embedded-postgres:2.1.1'
 
-    dependency 'com.github.ipld:java-cid:1.3.8'
+    dependency 'com.github.ipld:java-cid:1.3.10'
     dependency 'net.jodah:failsafe:2.4.4'
 
-    dependencySet(group: 'software.amazon.awssdk', version: '2.29.23') {
+    dependencySet(group: 'software.amazon.awssdk', version: '2.35.10') {
       entry 'bom'
       entry 'auth'
       entry 'secretsmanager'
       entry 'sts'
       entry 'kms'
     }
 
-    dependency 'io.rest-assured:rest-assured:5.5.0'
-
-    /*
-      com.squareup.okhttp3:logging-interceptor:4.9.0 // CVE-2021-0341
-      \--- org.web3j:core:4.9.2
-     */
-    dependency 'com.squareup.okhttp3:logging-interceptor:4.12.0'
-
     dependencySet(group: 'com.squareup.okhttp3', version: '4.12.0') {
       entry 'okhttp'
       entry 'mockwebserver'
+      entry 'logging-interceptor'   // CVE-2021-0341 via org.web3j:core:4.9.2
```

Comment:
is this CVE comment still relevant?

Reply:
It is. Web3J brings in older version of this library, hence we are overriding it to latest. I simply deleted the comment from a previous line and move it into dependencySet.